extensions/amp-analytics/0.1/requests.js

```diff
@@ -133,10 +141,35 @@ export class RequestHandler {
 
     if (!this.baseUrlPromise_) {
       expansionOption.freezeVar('extraUrlParams');
+
+      // expand requestOrigin if it is declared
+      if (this.requestOrigin_) {
```

Comment:
nit: `this.baseUrlPromise_` could got reset. I remember our decision is not to reset `this.requestOrigin_`. In that case, we may want to check for the existence of `this.requestOriginPromise_` independent of the existence of `this.baseUrlPromise_`